cmd/dt/unwrap.go

```diff
@@ -203,18 +210,25 @@ func pushChart(ctx context.Context, chart *chartutils.Chart, pushChartURL string
 	if err != nil {
 		return fmt.Errorf("failed to upload Helm chart: failed to create temp directory: %w", err)
 	}
-	tempTarFile := filepath.Join(dir, fmt.Sprintf("%s.tgz", chart.Name()))
-	if err := utils.Tar(chartPath, tempTarFile, utils.TarConfig{
+
+	tarCfg := utils.TarConfig{
 		Prefix: chart.Name(),
-		Skip: func(f string) bool {
+	}
+
+	if _, isLegacyBundle := wrap.(*chartutils.Chart); isLegacyBundle {
+		// If we are unwrapping a legacy bundle, the chart directory contains the artifacts/ and images/
```

Comment:
Could we assume that if anyone needs to unwrap legacy bundles they can just go and use an old release of `helm-dt`? Or simply wrap again, and unwrap...

I don't think this tool is so spread that it is worth to keep the code for handling legacy bundles.

Alternatively, we can always cleanup in the next release if we want to. But I dont think is worth, honestly.

Reply:
This comment actually applies to the rest of the PR where you're doing legacy handling logic. I don't think it is worth because users have alternatives: old release or re-wrap. And this tool is not so widely used, yet.